docs/book/component-guide/model-deployers/huggingface.md

```diff
@@ -72,9 +109,9 @@ Within the `HuggingFaceServiceConfig` you can configure:
 * `namespace`: The namespace where the Inference Endpoint will be created. The same namespace can be passed used while registering the Hugging Face model deployer.
 * `endpoint_type`: (Optional) The type of the Inference Endpoint, which can be `"protected"`, `"public"` (default) or `"private"`.
 
-For more information and a full list of configurable attributes of the Hugging Face Model Deployer, check out the [SDK Docs](https://sdkdocs.zenml.io/latest/integration\_code\_docs/integrations-huggingface/#zenml.integrations.huggingface.model\_deployers) and Hugging Face endpoint [code](https://github.com/huggingface/huggingface\_hub/blob/5e3b603ccc7cd6523d998e75f82848215abf9415/src/huggingface\_hub/hf\_api.py#L6957).
+For more information and a full list of configurable attributes of the Hugging Face Model Deployer, check out the [SDK Docs](https://sdkdocs.zenml.io/0.66.0/integration_code_docs/integrations-huggingface/) and Hugging Face endpoint [code](https://github.com/huggingface/huggingface_hub/blob/5e3b603ccc7cd6523d998e75f82848215abf9415/src/huggingface_hub/hf_api.py#L6957).
 
-### Run inference on a provisioned inference endpoint
+### Details on how to run inference on a provisioned inference endpoint
```

Comment:
I think `Run` is more concise, and that this section has details about running is already implied.